rust-deps/packages.bzl

```diff
@@ -12,9 +12,7 @@ RTTI_PACKAGES = {
         version = "1.0",
     ),
     "serde": crate.spec(
-        # TODO: Restore to "1.0" when https://github.com/bazelbuild/rules_rust/issues/2071
-        # is resolved
-        version = "=1.0.171",
+        version = "=1.0.184",
```

Comment:
Let's have this as just `version = "1.0",` otherwise serde will be pinned to version 1.0.184 instead.

Reply:
Ah, yep, that was the intent but the file didn't save.

Reply:
Also worth running repin again, `1.0.185` is already available.